### ISSUE TITLE
How to use torch.nn.SyncBatchNorm and torch.uitls.checkpoint together

### ISSUE THREAD

**Issue by devilztt** [pytorch/pytorch#43946]
When I use  net = torch.nn.SyncBatchNorm.convert_sync_batchnorm(net) to convert model,and  use torch.uitls.checkpoint in model ,loss backward, It seems to be stuck in process communication
